servers/rendering/rendering_device.cpp

```diff
@@ -645,6 +645,19 @@ Vector<uint8_t> RenderingDevice::buffer_get_data(RID p_buffer, uint32_t p_offset
 	return buffer_data;
 }
 
+uint64_t RenderingDevice::buffer_get_device_address(RID p_buffer) {
+	ERR_RENDER_THREAD_GUARD_V(0);
+
+	Buffer *buffer = _get_buffer_from_owner(p_buffer);
+	if (!buffer) {
+		ERR_FAIL_V_MSG(0, "Buffer argument is not a valid buffer of any type.");
```

Comment:
You can use `ERR_FAIL_COND_V_MSG(!buffer, 0, "Buffer ...");` on this line and below :)